doc/interfaces/reconstruction_flow.rst

```diff
@@ -0,0 +1,269 @@
+
+This command will save the DTI metrics to the specified output directory.
+
+--------------------------
+Diffusion Kurtosis Imaging
+--------------------------
+
+The diffusion kurtosis model is an expansion of the diffusion tensor model. In
+addition to the diffusion tensor (DT), the diffusion kurtosis model quantifies
+the degree to which water diffusion in biological tissues is non-Gaussian using
+the kurtosis tensor (KT) [Jensen2005]_.
+
+Measurements of non-Gaussian diffusion from the diffusion kurtosis model are of
+interest because they can be used to charaterize tissue microstructural
+heterogeneity [Jensen2010]_. 
+
+Moreover, DKI can be used to:
+
+* Derive concrete biophysical parameters, such as the density of axonal fibers and diffusion tortuosity [Fierem2011]_
+
+* Resolve crossing fibers in tractography and to obtain invariant rotational measures not limited to well-aligned fiber populations [NetoHe2015]_.
+
+We will use the ``cfin_multib`` dataset in DIPY to showcase this reconstruction
+method. You can also use your own data!
+
+Create the directories in which to save the DKI metrics (e.g.: ``recons_dki_output``)
+and mask file (e.g.: ``cfin_multib_mask``)::
+
+    mkdir recons_dki_output cfin_multib_mask
+
+The Diffusion Kurtosis Imaging reconstruction method requires the paths to the
+diffusion input file, b-values file, b-vectors file and mask file. The optional
+parameters can also be provided. In this case, we will be specifying the threshold
+used to find b=0 directions (``b0_threshold``) and the output directory (``out_dir``). 
+
+To get the mask file, run the command as::
+
+    dipy_median_otsu data/cfin_multib/__DTI_AX_ep2d_2_5_iso_33d_20141015095334_4.nii --vol_idx 3 4 5 6 --out_dir "cfin_multib_mask"
+
+To run the DKI reconstruction method on the data, execute the ``dipy_fit_dki``
+command, e.g.::
+
+    dipy_fit_dki data/cfin_multib/__DTI_AX_ep2d_2_5_iso_33d_20141015095334_4.nii data/cfin_multib/__DTI_AX_ep2d_2_5_iso_33d_20141015095334_4.bval data/cfin_multib/__DTI_AX_ep2d_2_5_iso_33d_20141015095334_4.bvec cfin_multib_mask/brain_mask.nii.gz --b0_threshold 70.0 --out_dir recons_dki_output
+
+This command will save the DKI metrics to the specified output directory.
+
+--------------------
+Constant Solid Angle
+--------------------
+
+We are using ``stanford_hardi`` dataset. As with any other workflow in DIPY, you can
+also use your own data!
+
+Create a directory in which to save the peaks volume (e.g.: ``recons_csa_output``)::
+
+    mkdir recons_csa_output
+
+The workflow for the CSA reconstruction method requires the paths to the
+diffusion input file, b-values file, b-vectors file and mask file. The optional
+arguments can also be provided. In this case, we will be specifying whether or
+not to save pam volumes as single nifti files (``extract_pam_values``) and the
+output directory (``out_dir``).
+
+To perform the CSA reconstruction, we will run the command as::
```

Comment:
Similar comment as above concerning mentioning the command.

Reply:
Addressed in 4ce067c